### ISSUE TITLE
Juniper Attributes Problem When Using Regex

### ISSUE THREAD

**Issue by mhenphyz** [tacacsgui/tacacsgui#102]
**Describe the system**
Ubuntu version: 
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.5 LTS
Release:        18.04
Codename:       bionic

PHP version: 
PHP 7.3.24-3+ubuntu18.04.1+deb.sury.org+1 (cli) (built: Oct 31 2020 16:59:59) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.24, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.24-3+ubuntu18.04.1+deb.sury.org+1, Copyright (c) 1999-2018, by Zend Technologies

TacacsGUI API version: **IMPORTANT** 
0.9.82

**Describe**

Actually, this is not a bug, but a change suggestion, recently I had a problem with users authentication on juniper devices where the output of commands was very slow using some regex on the authorization attributes. Then juniper engineer recommended changing the attributes from "allow/deny-configuration" to "allow-configuration-regexps and deny-configuration-regexps". After that, the problem is gone.  Maybe is a good idea to change the juniper service pattern to use these two attributes by default.

You can check that on juniper kb.

https://kb.juniper.net/InfoCenter/index?page=content&id=KB27920

**Comment by mhenphyz** [comment]
If you agree I created the pull request to change the attribute.  #103

**Comment by gurkin33** [comment]
Hello,

It is a good idea, but "...-configuration-regexps" is something new and was released in version Junos OS 11.2.
We have to check this attribute.
Any way you can set it manually in a service.

Regards, Alexey

**Comment by mhenphyz** [comment]
Of course, I agree with you, but for information, the end of life of version 11.2 was in 2013,

https://support.juniper.net/support/eol/software/junos/
